checks.d/haproxy.py

```diff
@@ -338,11 +362,12 @@ def _process_status_metric(self, hosts_statuses, collect_status_metrics_by_host,
                     self.warning('`collect_status_metrics_by_host` is enabled but no host info\
                                  could be extracted from HAProxy stats endpoint for {0}'.format(service))
                 service, status = host_status
-            status = status.lower()
 
             if self._is_service_excl_filtered(service, services_incl_filter, services_excl_filter):
                 continue
 
+            status = self._normalize_status(status)
```

Comment:
applying it in `_line_to_dict` would also require changing the status comparisons that are done in `_process_event`. Not necessarily complex though, I'll have a closer look to see what could be done there